server/src/routes/series.js

```diff
@@ -0,0 +1,126 @@
+const express = require('express');
+const mongoose = require("mongoose");
+const HttpStatus = require("http-status-codes");
+const Series = require('../db/mongo/models/series');
+const Genres = require('../db/mongo/models/genre');
+
+const { filterSeries, getMostWatchedSeries, aggregateSeries, getCommonSeriesAmongFollowing } = require('../services/series');
+const { getUserWatchlist } = require('../services/watchlist');
+
+const router = express.Router();
+
+const pageLimit = 10;
+
+router.get('/', async (req, res) => {
+    const { pageNumber } = req.query;
+
+    let response;
+    let statusCode = HttpStatus.OK;
+    try {
+        response = await filterSeries(req.body, pageNumber, pageLimit);
```

Comment:
Data like filters should be retrieved from `req.query`. GET requests shouldn't contain body.
You can merge it with `pageNumber` that's already retrieved from `req.query`